test/tests/menubar-2.js

```diff
@@ -0,0 +1,1086 @@
+const confirmAttributeValue = require('../util/confirmAttributeValue');
+const confirmRovingTabindex = require('../util/confirmRovingTabindex');
+
+const exampleFile = 'menubar/menubar-2/menubar-2.html';
+
+let ex = {
+  menubarSelector: '#ex1 [role="menubar"]',
+  menubarMenuitemSelector: '#ex1 a[role="menuitem"]',
+  submenuMenuitemSelector: '#ex1 li[role="menuitem"]',
+  submenuSelector: '#ex1 [role="menu"]',
+  menuitemcheckboxSelector: '#ex1 [role="menuitemcheckbox"]',
+  groupSelector: '#ex1 [role="group"]',
+  menuitemradioSelector: '#ex1 [role="menuitemradio"]',
+  numMenus: 4,
+  numSubmenuItems: [4, 10, 4, 7],
+  allSubmenuItems: [
+    '#ex1 [role="menu"][aria-label="Font"] li[role="menuitemradio"]',
+    '#ex1 [role="menu"][aria-label="Style/Color"] li[role="menuitemradio"],#ex1 [role="menu"][aria-label="Style/Color"] li[role="menuitemcheckbox"]',
+    '#ex1 [role="menu"][aria-label="Text Align"] li[role="menuitemradio"]',
+    '#ex1 [role="menu"][aria-label="Size"] li[role="menuitem"],#ex1 [role="menu"][aria-label="Size"] li[role="menuitemradio"]'
+  ],
+  radioItemGroupings: [
+    {
+      menuIndex: 0,
+      itemsSelector: '#ex1 [role="menu"][aria-label="Font"] [role="menuitemradio"]',
+      defaultSelectedIndex: 0
+    },
+    {
+      menuIndex: 1,
+      itemsSelector: '#ex1 [role="group"][aria-label="Text Color"] [role="menuitemradio"]',
+      defaultSelectedIndex: 0
+    },
+    {
+      menuIndex: 1,
+      itemsSelector: '#ex1 [role="group"][aria-label="Text Decoration"] [role="menuitemradio"]',
+      defaultSelectedIndex: 0
+    },
+    {
+      menuIndex: 2,
+      itemsSelector: '#ex1 [role="menu"][aria-label="Text Align"] [role="menuitemradio"]',
+      defaultSelectedIndex: 0
+    },
+    {
+      menuIndex: 3,
+      itemsSelector: '#ex1 [role="group"][aria-label="Font Sizes"] [role="menuitemradio"]',
+      defaultSelectedIndex: 2
+    }
+  ]
+};
+
+const exampleInitialized = async function (t) {
+  let initializedSelector = ex.menubarMenuitemSelector + '[tabindex="0"]';
+
+  await t.context.session.wait(async function () {
+    let els = await t.context.session.findElements(By.css(initializedSelector));
+    return els.length === 1;
+  }, 100);
+};
+
+const checkmarkVisible = function (/* selector, index*/) {
+  const [selector, index] = arguments;
+  let checkmarkContent = window.getComputedStyle(
+    document.querySelectorAll(selector)[index], ':before'
+  ).getPropertyValue('content');
+  if (checkmarkContent == 'none') {
+    return false;
+  }
+  return true;
+};
+
+const checkFocus = function (/* selector, index*/) {
+  const [selector, index] = arguments;
+  let items = document.querySelectorAll(selector);
+  return items[index] === document.activeElement;
+};
+
+
+// Attributes
+
+ariaTest('Test for role="menubar" on ul', exampleFile, 'menubar-role', async (t) => {
+
+  t.plan(2);
+
+  let menubars = await t.context.session.findElements(By.css(ex.menubarSelector));
```

Comment:
`const`, the variable is never reassigned. Goes for any `let` that never gets reassigned.